### PR TITLE
Add provider TV-Vault

### PR DESCRIPTION
**Pull request by duramato**

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked
- [ ] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

File: sickbeard/providers/tvvault.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    def seed_ratio(self):
+        return self.ratio
+
+
+class TvVaultCache(tvcache.TVCache):
+    def __init__(self, provider_obj):
+        tvcache.TVCache.__init__(self, provider_obj)
+
+        # Only poll TransmitTheNet every 20 minutes max
+        self.minTime = 0.5
```
Comment (medariox) [inline sickbeard/providers/tvvault.py:181]: The tracker is going to love this! :P

Comment (duramato) [inline sickbeard/providers/tvvault.py:181]: Its for testing! lol

**Comment by adaur** [comment]
Need help?

**Comment by duramato** [comment]
@adaur feel free to continue it as i`ve hit a dead end

**Comment by savahu** [comment]
This would be really nice for old TV series in Sickrage..

**Pull request closed without merging**